dojo1

austincqdo opened the pull request.

austincqdo committed:
added pseudo file to dev branch

austincqdo committed:
removed pseudo file, readded commented files

austincqdo committed:
added 1-1 1-2 1-3

austincqdo commented:
Hi Masa-san, the files from the first exercise are in 3-4/3-4.js, 3-5/3-5.js, and 3-7/3-7.js. The ones from the second exercise are in the folders 1-1, 1-2, 1-3. It is my first time using the pull request w/ review feature on github so forgive me if I made a mistake m(*-ω-)m

mmiyahara commented on 1-1/1-1.js:
```diff
@@ -0,0 +1,32 @@
+const fileManage = {
+    'f001': {'Folder Name': 'Information System Department',
+        'View Auth': 'Information System Department',
```
```
    'f001': {'Folder Name': 'Information System Department',
        'View Auth': 'Information System Department',
        // ...
```

It's just a matter of taste, but I personally prefer a format like this.
```
    'f001': {
        'Folder Name': 'Information System Department',
        'View Auth': 'Information System Department',
        // ...
```

mmiyahara commented on 1-3/1-3.js:
```diff
@@ -0,0 +1,55 @@
+
+const list = document.getElementById('list');
+for (let i = 0; i < allAccess.length; i++) {
+    const node = document.createElement('LI');
```
We normally use lowercase instead of uppercase for a tag name.

> **tagName**
> When called on an HTML document, createElement() converts tagName to lower case before creating the element.
[Document.createElement() - Web APIs | MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement#Parameters)

According to MDN, `createElement()` converts tagName to lowercase before creating an element. But there is no reason to make it uppercase.

mmiyahara replied:
Let's change the name of the variable `node` to a more meaningful one such as `li` for readability.

mmiyahara commented on 1-3/1-3.js:
```diff
@@ -0,0 +1,55 @@
+    }
+}
+
+const list = document.getElementById('list');
```
Let's make the variable name more meaningful such as `ul` as `document.getElementById('list')` should refer to a `ul` element.

mmiyahara commented on 3-4/3-4.js:
```diff
@@ -1,3 +1,4 @@
+// Guessing
```
Remove these files from the `master` branch as all the codes in the `master` branch should be placed after being reviewed.

mmiyahara replied:
I can't put comments after line 5 as it has already been placed in the `master` branch somehow. Let me put comments there after you delete them from the `master` branch.

mmiyahara commented on 3-5/3-5.js:
```diff
@@ -1,3 +1,4 @@
+// Dates
```
Remove these files from the master branch as all the codes in the master branch should be placed after being reviewed.

mmiyahara replied:
I can't put comments after line 5 as it has already been placed in the master branch somehow. Let me put comments there after you delete them from the master branch.

mmiyahara commented on 3-7/3-7.js:
```diff
@@ -1,3 +1,4 @@
+// Battle
```
Remove these files from the master branch as all the codes in the master branch should be placed after being reviewed.

mmiyahara replied:
I can't put comments after line 5 as it has already been placed in the master branch somehow. Let me put comments there after you delete them from the master branch.

mmiyahara commented on 3-4/3-4.js:
```diff
@@ -1,3 +1,4 @@
+// Guessing
 var num = window.prompt("Enter a number");
```
Use `let` or `const` instead of `var`.

mmiyahara reviewed (changes requested):
- Remove the folders `3-4`, `3-5`, and `3-7` from the `master` branch!
- Don't forget to include the folders `3-4`, `3-5`, and `3-7` in the `dev` branch when submitting a pull request again.
- I gave you feedback. Fix your codes based on it and submit a pull request again.

austincqdo committed:
adding 2-1, 2-2, 2-3 to dev

austincqdo committed:
added 1-1 1-2 1-3 3-4 3-5 3-7

austincqdo commented:
Hi Masa-san, I removed 3-4, 3-5, and 3-7 from the master branch, revised 1-1, 1-2, 1-3, 3-4, 3-5, and 3-7 as you outlined, and added 2-1, 2-2, and 2-3 (.html files) for review. Thanks!

mmiyahara commented on 3-4/3-4.js:
```diff
@@ -1,7 +1,6 @@
 if (num && parseInt(num, 10) <= 5 && parseInt(num, 10) >= 0) {
-	var random = Math.floor(6 * Math.random());
+	const random = Math.floor(6 * Math.random());
 	if (num == random) {
```
Use `===` instead of `==` as much as possible.
[JavaScript Showdown: == vs === - codeburst](https://codeburst.io/javascript-showdown-vs-7be792be15b5)

mmiyahara reviewed (approved):
Great!

mmiyahara commented:
@austincqdo 
Please merge into the `master` branch when ready.

austincqdo committed:
changed == to ===

austincqdo committed:
added 5-5 slideshow

austincqdo committed:
added border to slideshow

austincqdo commented:
Added 5-5, slideshow.

austincqdo committed:
removed text highlighting issue in 5-5

austincqdo committed:
added 3-3 weather api

austincqdo commented:
Added commit for 3-3, SF Weather.
